lib/studies/reply/adaptive_nav.dart

```diff
@@ -726,6 +724,10 @@ class _AnimatedBottomAppBar extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    var fadeOut = Tween<double>(begin: 1, end: -1).animate(
```

Comment:
Why does this go to -1?

Reply:
I do this so the animation moves backwards instead of forwards. FadeTransition doesn't have a way to specify a reverse, so our end < begin. 

Also it's -1 because we want the mailbox title to be invisible by the time the drawer expands half way. So by setting this to -1 at the halfway point the opacity will be 0.